MSALiOSB2C/ViewController.swift

```diff
@@ -90,7 +92,10 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
          */
         
         do {
-            let application = try MSALPublicClientApplication.init(clientId: kClientID, authority: kAuthority)
+            let authority = try MSALB2CAuthority.init(url: URL(string:kAuthority)!)
```

Comment:
This PR is now outdated regarding new MSAL public API surface, but I liked how the URL was constructed here: https://github.com/Azure-Samples/active-directory-b2c-ios-swift-native-msal/pull/13/files

Reply:
addressed :)